src/excalidraw-app/data/localStorage.ts

```diff
@@ -127,3 +129,35 @@ export const getTotalStorageSize = () => {
     return 0;
   }
 };
+
+export const savePublishLibDataToStorage = (data: PublishLibraryDataParams) => {
+  try {
+    localStorage.setItem(
+      STORAGE_KEYS.LOCAL_STORAGE_KEY_PUBLISH_LIBRARY,
+      JSON.stringify(data),
+    );
+  } catch (error: any) {
+    // Unable to access window.localStorage
+    console.error(error);
+  }
+};
+
+export const importPublishLibDataFromStorage = () => {
+  try {
+    const data = localStorage.getItem(
+      STORAGE_KEYS.LOCAL_STORAGE_KEY_PUBLISH_LIBRARY,
+    );
+    if (data) {
+      return JSON.parse(data);
+    }
+  } catch (error: any) {
+    // Unable to access localStorage
+    console.error(error);
+  }
+
+  return null;
+};
+
+export const clearPublishLibDataInStorage = () => {
+  localStorage.removeItem(STORAGE_KEYS.LOCAL_STORAGE_KEY_PUBLISH_LIBRARY);
+};
```

Comment:
this is an `excalidraw-app` module. So either inline it into the `PublishLibrary.tsx` file, or move the LS logic out of the editor completely. But I think this _should_ be done within the editor.

Reply:
good point, I will move it to `PublishLibrary`